### ISSUE TITLE
[Feature]: Increase no of fields in JSON form from 50

### ISSUE THREAD

**Issue by dilippitchika** [appsmithorg/appsmith#13165]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Creating this issue to track why we should increase the number of items in JSON form from 50

### Why should this be worked on?

Needs more validation

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_5r364)

**Comment by dilippitchika** [comment]
Conversations

https://app.intercom.com/a/apps/y10e7138/inbox/inbox/conversation/164629100091509

**Comment by dilippitchika** [comment]
No instrumentation exists in this today for me to determine the reach. @ashit-rath should we log a debugger error message event for this case? I don't think we need to add a new event.

**Comment by geekyme-fsmk** [comment]
+1 on this feature request. Loan systems will typically have >50 fields. See this for an example of a loan retrieval call - https://demo.mifos.io/api-docs/apiLive.htm#loans_retrieve

**Comment by nirkons** [comment]
+1 for me, we have an object with upwards of 100 fields..
kind of strange that there is this hardcoded limit :(

workaround:
either i have to split it into 2 json forms or I need to make all the fields manually one by one :/

**Comment by rizistt** [comment]
+1 for me too.

**Comment by okandemirofficial** [comment]
I have also facing this issue

**Comment by nirkons** [comment]
Still waiting on this, 
+ I dont know if to open another issue for a feature request but there are some field types such as Rich Text Editor and maybe some field options missing from JSON forms which is surely needed.

**Comment by Ssillencer** [comment]
+1 I constantly hit the limit

**Comment by ssciacco** [comment]
+1 
not sure why this limit is not configurable and is set so low. 
if you have an array inside the json each with a few fields the limit make json for unusable

**Comment by marcjaner** [comment]
still facing the same issue here

**Comment by nirkons** [comment]
> +1 for me, we have an object with upwards of 100 fields.. kind of strange that there is this hardcoded limit :(
> 
> workaround: either i have to split it into 2 json forms or I need to make all the fields manually one by one :/

Over a year has gone by, still no change :(
